src/Features/ForgetPassword/ducks.js

```diff
@@ -0,0 +1,8 @@
+export const ForgetPasswordReducer = (state = {}, action) => {
+  const { type } = action;
+  switch (type) {
+    case 'ForgetPassword': // must Proper const here
```

Comment:
You should not be use this comment instead use proper constants or removed the case ..